go/client/chat_cli_alias_test.go

```diff
@@ -0,0 +1,36 @@
+package client
+
+import (
+	"testing"
+
+	"github.com/keybase/client/go/protocol/chat1"
+	"github.com/stretchr/testify/require"
+)
+
+func mustMakeConvID(t *testing.T, val string) chat1.ConversationID {
+	id, err := chat1.MakeConvID(val)
+	require.NoError(t, err)
+	return id
+}
+
+func TestConversationAliasPrefixNumOfBytes(t *testing.T) {
+	prefixNumOfBytes, err := ConversationAliasPrefixNumOfBytes([]FullConversationAlias{
+		MakeFullConversationAlias(mustMakeConvID(t, "00000100000000000000")),
+		MakeFullConversationAlias(mustMakeConvID(t, "00000200000000000000")),
+	}, 2)
+	require.NoError(t, err)
+	require.Equal(t, 2, prefixNumOfBytes)
+
+	prefixNumOfBytes, err = ConversationAliasPrefixNumOfBytes([]FullConversationAlias{
+		MakeFullConversationAlias(mustMakeConvID(t, "00000000100000000000")),
+		MakeFullConversationAlias(mustMakeConvID(t, "00000000200000000000")),
+	}, 2)
+	require.NoError(t, err)
+	require.Equal(t, 3, prefixNumOfBytes)
+
+	prefixNumOfBytes, err = ConversationAliasPrefixNumOfBytes([]FullConversationAlias{
+		MakeFullConversationAlias(mustMakeConvID(t, "00000000000000000000")),
+		MakeFullConversationAlias(mustMakeConvID(t, "00000000000000000000")),
+	}, 2)
+	require.Error(t, err)
+}
```

Comment:
There's a case here checking the situation that requires more than minimum bytes, but not the actual alias content (i.e. only the number of bytes). I'll add one to check content as well